### DISCUSSION TITLE
DateOnly? deserialization

### DISCUSSION THREAD

**Discussion by rbobro** [ServiceStack/Discuss#115] [Q&A]
In Servicestack.Text nullable DateOnly and TimeOnly types with null values are deserialized to default values instead of null values. Can this be fixed ?

https://github.com/ServiceStack/ServiceStack/blob/681b152271750ad89e59a554a2c12f44cbdcf79d/ServiceStack.Text/src/ServiceStack.Text/Common/DeserializeBuiltin.cs#L130

**Comment by mythz** [comment]
should be resolved from [this commit](https://github.com/ServiceStack/ServiceStack/commit/c2f432d6fa839ecf5b6e11491513da9c4236f330). 

This change is available from v8.2.3+ which is now available in [pre release packages](https://docs.servicestack.net/pre-release).
